spec/models/principal_role_spec.rb

```diff
@@ -35,17 +33,19 @@
       @principal_role = FactoryGirl.build(:principal_role)
     end
 
-    describe "role not assignable to user" do
+    describe 'role not assignable to user' do
       before :each do
         allow(@principal_role.role).to receive(:assignable_to?).and_return(false)
       end
 
       it { expect(@principal_role.valid?).to be_falsey }
-      it { @principal_role.valid?
-           expect(@principal_role.errors[:base]).to include(I18n.t(:error_can_not_be_assigned))}
+      it {
+        @principal_role.valid?
+        expect(@principal_role.errors[:base]).to include(I18n.t(:error_can_not_be_assigned))
+      }
     end
 
-    describe "role assignable to user" do
+    describe 'role assignable to user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.